src/main/resources/assets/entropy/lang/en_us.json

```diff
@@ -162,6 +162,7 @@
   "events.entropy.rainbow_sheep_everywhere": "jeb_",
   "events.entropy.armor_trim": "Trimmed",
   "events.entropy.bouncy_blocks": "Bouncy Blocks",
+  "events.entropy.racing_balloon": "Racing Balloon",
```

Comment:
Suggestion: "Balloon Race" might be a more fun name

Reply:
I feel like that implies there is someone else to race against, which there isn't in singleplayer. I'm not entirely opposed to it though.

Reply:
IMO both names conveys a similar feeling to me. I think i'm a bit more inclined towards Ballon Race... somehow (as a non native speaker) sounds better hahaha.